packages/base-styles/_mixins.scss

```diff
@@ -659,7 +670,7 @@
 	outline-color: var(--wp-admin-theme-color);
 	outline-style: solid;
 	outline-width: calc(#{$widthRatio} * (var(--wp-admin-border-width-focus) / var(--wp-block-editor-iframe-zoom-out-scale, 1)));
-	outline-offset: calc(#{$widthRatio} * ((-1 * var(--wp-admin-border-width-focus) ) / var(--wp-block-editor-iframe-zoom-out-scale, 1)));
+	outline-offset: calc(#{$widthRatio} * ((-1 * var(--wp-admin-border-width-focus)) / var(--wp-block-editor-iframe-zoom-out-scale, 1)));
 }
 
 @mixin selected-block-focus($widthRatio: 1) {
```

Comment:
Are all these random CSS changes related to this PR?

Reply:
Looks like a lot of formatting changes got in, when fixing the mixins at the top of this file.

Reply:
Fixed: https://github.com/WordPress/gutenberg/pull/67390/commits/a0c5fa9d754bd7772d86c1dbecaea52e3fffb71c